sdk/core/Azure.Core.TestFramework/src/ClientTestFixtureAttribute.cs

```diff
@@ -156,6 +157,13 @@ private void ProcessTest(object serviceVersion, bool isAsync, int serviceVersion
                 test.Properties.Add("SkipRecordings", $"Test is ignored when not running live because the service version {serviceVersion} is not the latest.");
             }
 
+            if ("true" == Environment.GetEnvironmentVariable(UseLatestServiceVersionKey) && serviceVersionNumber != _maxServiceVersion)
+            {
+                test.RunState = RunState.Ignored;
+                test.Properties.Set("_SKIPREASON",
+                    $"Test is ignored when running live because the service version {serviceVersion} is not the latest while latest service version has been requested.");
```

Comment:
Let's point folks to why it was ignored.  `$"Test ignored because {UseLatestServiceVersionKey} is set in the environment and version {serviceVersion} is not the latest."`